docker-compose.yml

```diff
@@ -1,4 +1,4 @@
-version: '3'
+version: '3.8'
```

Comment:
I suggest we keep this at `3` so we don't require the latest version as `condition` is supported anyway.